vignettes/EpiNow2.Rmd.orig

```diff
@@ -95,7 +95,7 @@ estimates <- epinow(
   generation_time = gt_opts(example_generation_time),
   delays = delay_opts(example_incubation_period + reporting_delay),
   rt = rt_opts(prior = LogNormal(mean = 2, sd = 0.2)),
-  stan = stan_opts(cores = 4, control = list(adapt_delta = 0.99)),
+  stan = stan_opts(cores = 4, control = list(adapt_delta = 0.9)),
```

Comment:
There's a mismatch between the original and rendered versions. Was this changed again?

Reply:
its from the merging from main I thinkl